engine/buff/buff.cpp

```diff
@@ -620,13 +620,16 @@ buff_t::buff_t( sim_t* sim, player_t* target, player_t* source, util::string_vie
     is_fallback(),
     requires_invalidation(),
     expire_at_max_stack(),
+    ignore_mod_time_modifier( false ),
```

Comment:
This uses "modifier" twice. Same as above, I'd call it something like `ignore_time_multiplier`.